release-notes.md

```diff
@@ -1,3 +1,18 @@
+# Release 1.9.5
```

Comment:
I didn't see the version file changed in this pr. I assume that's been done elsewhere